script/command_functions.lua

```diff
@@ -137,7 +137,8 @@ me.reacquire_ctrons = function()
     for s, surface in pairs(game.surfaces) do
         global.constructrons_count[surface.index] = 0
         local constructrons = surface.find_entities_filtered {
-            name = {"constructron", "constructron-rocket-powered"},
+            name = {"constructron", "constructron-rocket-powered",
+            "constructron-ss-space-spidertron", "constructron-spidertronmk2", "constructron-spidertronmk3"},
```

Comment:
This should also be a global list, which will make this much more readable

``` lua
-- somewhere else
local valid_constructrons = {
    "constructron", 
    "constructron-rocket-powered", 
    "constructron-ss-space-spidertron", 
    "constructron-spidertronmk2", 
    "constructron-spidertronmk3",
}

```
```suggestion
            name = valid_constructrons,
```